apps/flux-system/base/image-automation-components.yaml

```diff
@@ -2005,7 +2047,7 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        image: hmctspublic.azurecr.io/fluxcd/image-automation-controller:v0.19.0
+        image: ghcr.io/fluxcd/image-automation-controller:v0.26.1
```

Comment:
is this intentionally changing? it should be fine but just want to make sure its intended

Reply:
Spoke with @adusumillipraveen, we were unsure why its imported into registry, checked against sds and they are using source